languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml

```diff
@@ -21187,4 +21187,552 @@ Pour en savoir plus sur les tone tags, suivez le lien vers notre documentation (
             <example correction="ainsi|alors|en conséquence|de ce fait">Le vin blanc est <marker>donc</marker> idéal pour accompagner les viandes blanches.</example>
         </rule>
     </category>
+    <category id="PONCTUATION_STYLE_OS" name="Ponctuation style OS">
```

Comment:
praise: that's a good moove 🥜

Reply:
We can get rid of the loop by deleting the second suggestion (which doesn't really make sense anyway, or I'm just not seeing something):

```
            <rule tags="picky">
                <pattern>
                    <token postag="V.* (ind|con|sub).*" postag_regexp="yes">
                        <exception postag="[JN].*" postag_regexp="yes"/>
                    </token>
                    <marker>
                        <token>,</token>
                        <token postag="A">
                            <exception regexp="yes">en|par|non|outre|hier|sinon|oui|à|etc|ex|même|d'|sans|là|ici|ne|n'|voire|bien|soit</exception></token>
                        <token>,</token>
                    </marker>
                </pattern>
                <message>La virgule peut être omise.</message>
                <suggestion> \3</suggestion>
                <suggestion> \3,</suggestion>
                <example correction=" notamment| notamment,">Il ne vient<marker>, notamment,</marker> le mardi.</example>
            </rule>
```

The idea would be to delete: <suggestion> \3,</suggestion>

Can you see any issue with this deletion @LucieSteib?

Reply:
OOoh, yeah, that will do the trick, yes! 
Worst case we have to do something after tonight's diff.